templates/node/lib/client.js.twig

```diff
@@ -109,8 +109,12 @@ class Client {
             return response.data;
         } catch(error) {
             if('response' in error) {
-                if('data' in error.response) {
-                    throw new {{spec.title | caseUcfirst}}Exception(error.response.data.message, error.response.status, error.response.data);
+                if(error.response && 'data' in error.response) {
```

Comment:
This was found because of the issue raised in discord
https://discord.com/channels/564160730845151244/812714347298160690/840920049635688480